src/gui/dialogs/reachmap_options.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+	Copyright (C) 2023 - 2024
+	Part of the Battle for Wesnoth Project https://www.wesnoth.org/
+
+	This program is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; either version 2 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY.
+
+	See the COPYING file for more details.
+*/
+
+#pragma once
+
+#include "gui/dialogs/modal_dialog.hpp"
+#include "gui/widgets/group.hpp"
+
+#include <map>
+
+namespace gui2::dialogs
+{
+/**
+ * @ingroup GUIWindowDefinitionWML
+ *
+ * A test dialog for testing various gui2 features
```

Comment:
This looks like copy&paste from somewhere that was not adjusted.

Reply:
yes, this is a leftover from early development of this PR when I used a copy of the test dialog as a base. will remove

Reply:
fixed